SnapMD.VirtualCare.ApiModels/SnapMD.VirtualCare.ApiModels.csproj

```diff
@@ -98,6 +98,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AddressLocationObject.cs" />
     <Compile Include="AddressObject.cs" />

```

Comment:
There is already an AddressObject, what is the difference between AddressLocationObject and AddressObject?